README.md

```diff
@@ -34,15 +35,21 @@ installs T2T, downloads MNIST, trains a model and evaluates it:
 ```
 pip install tensor2tensor && t2t-trainer \
   --generate_data \
-  --data_dir=~/t2t_data \
-  --output_dir=~/t2t_train/mnist \
+  --data_dir=./t2t_data \
+  --output_dir=./t2t_train/mnist \
   --problem=image_mnist \
   --model=shake_shake \
   --hparams_set=shake_shake_quick \
   --train_steps=1000 \
   --eval_steps=100
 ```
 
+### Run on FloydHub
```

Comment:
Could you move this section to the end, before papers? Thanks!

Reply:
Done!